tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -446,6 +477,10 @@ func getLog(runner *kubectlcmd.Runner, resources *cluster2.Resources, config *co
 
 func runAnalyze(config *config.BugReportConfig, params *content.Params, analyzeTimeout time.Duration) {
 	newParam := params.SetNamespace(common.NamespaceAll)
+	defer func(start time.Time) {
+		log.Infof("Time used for running istio analyze on all namespaces and report is", time.Since(start))
```

Comment:
There is no `%v` here to insert the time

Reply:
Also `istio` to `Istio`